src/main/java/org/openelisglobal/action/service/ActionServiceImpl.java

```diff
@@ -8,11 +8,13 @@
 
 @Service
 public class ActionServiceImpl extends AuditableBaseObjectServiceImpl<Action, String> implements ActionService {
-    @Autowired
-    protected ActionDAO baseObjectDAO;
 
-    ActionServiceImpl() {
+    private final ActionDAO baseObjectDAO;
```

Comment:
Thanks @git-huber-hubbing .
Fo consistence we always use property-based -Autowiring  not Contsructor based Autowriring